### PR TITLE
Fix initial credentials exchange

### PR DESCRIPTION
**Pull request by suda**

### CODE REVIEW

File: .github/workflows/push.yaml [hunk 1]
```diff
@@ -61,4 +61,4 @@ jobs:
           make install
       - name: Run bandit
         run: |
-          poetry run bandit -r ./py_ocpi
+          poetry run bandit -r ./py_ocpi -s B105
```
Comment (HamedAkhavan) [inline .github/workflows/push.yaml:64]: please don't turn off this security check

File: py_ocpi/core/utils.py [hunk 2]
```diff
@@ -18,7 +19,10 @@ def set_pagination_headers(response: Response, link: str, total: int, limit: int
 def get_auth_token(request: Request) -> str:
     headers = request.headers
     headers_token = headers.get('authorization', 'Token Null')
-    return headers_token.split()[1]
+    token = headers_token.split()[1]
+    if token == 'Null':
```
Comment (HamedAkhavan) [inline py_ocpi/core/utils.py:23]: you can disable the bandit security check for this line only

**Review by HamedAkhavan — CHANGES_REQUESTED** [review]

File: py_ocpi/core/utils.py (hunk 2 above)
Comment (suda) [reply py_ocpi/core/utils.py:23]: @HamedAkhavan do you have an example of how to do it? I only was able to find how to skip a whole test together

Comment (HamedAkhavan) [reply py_ocpi/core/utils.py:23]: change the line to `if token == 'Null':  # nosec`

**Review by HamedAkhavan — APPROVED** [review]

**Pull request merged**